client/src/document/ingredients/polyfills-subsection.tsx

```diff
@@ -0,0 +1,66 @@
+import { DisplayH3 } from "./utils";
```

Comment:
I'd call this file just `polyfills.tsx`, as the "SubSection" is already included in the `PolyfillsSubSection` component name.